btr-api/sample_data/load_data.py

```diff
@@ -125,28 +132,35 @@ def _get_nationalities():
                 'code': row['code'],
                 'name': row['name']
             })
+            if limit and len(nationalities.keys()) == limit:
+                return nationalities
     return nationalities
 
 
-def _get_addresses():
+def _get_addresses(limit: int = None):
     """Return the person addresses."""
     addresses: dict[str, list] = {}
     with open('bods_csvs/person_addresses.csv', encoding='utf-8') as person_address_csv:
         reader = csv.DictReader(person_address_csv)
         for row in reader:
+            country = row['country']
```

Comment:
So, are we storing the country name or 2 letter iso in the DB now ?
It would be good to have additional line (if we want flat data) that holds country 2 letter iso. 
As the name can be international, or in their own language sometimes. 

E.G. Hrvatska -> Croatia. And depending from the source, it could be different, while 2 letter iso code is always HR.

We can deal with this in another ticket if you agree.

Reply:
ah yes good point. I am storing the full name instead of the iso code here, but I agree with you. Will chat with you and probably put this in a new ticket

Reply:
alright updated the loader and I am rerunning it over dev. Created a ticket for the other address pieces https://github.com/bcgov/entity/issues/19919